src/wp-includes/class-wp-scripts.php

```diff
@@ -318,9 +318,12 @@ public function do_item( $handle, $group = false ) {
 			$before_script = sprintf( "<script%s id='%s-js-before'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $before_script );
 		}
 
-		$strategy     = $this->get_eligible_loading_strategy( $handle );
-		$after        = ( 'defer' === $strategy || 'async' === $strategy ) ? 'after-standalone' : 'after';
-		$after_script = $this->print_inline_script( $handle, $after, false );
+		$intended_strategy    = $this->get_data( $handle, 'strategy' );
+		$strategy             = $this->get_eligible_loading_strategy( $handle );
+		$strategy_has_changed = false !== $intended_strategy && '' !== $strategy && $intended_strategy !== $strategy;
+		$strategy_data_attr   = $strategy_has_changed ? $intended_strategy : $strategy;
+		$after                = ( 'defer' === $strategy || 'async' === $strategy ) ? 'after-standalone' : 'after';
```

Comment:
Note the `standalone` bit would be removed by https://github.com/10up/wordpress-develop/pull/67